calendar/index.html

```diff
@@ -1,36 +1,54 @@
+<!DOCTYPE html>
 <html lang="en">
-  <head>
-    <link rel="stylesheet" href="https://uicdn.toast.com/calendar/latest/toastui-calendar.min.css" />
-    <link rel="stylesheet" href="screen.css" />
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <script src="https://uicdn.toast.com/calendar/latest/toastui-calendar.min.js"></script>
-    <script src="https://docs.getgrist.com/grist-plugin-api.js"></script>
+<head>
+    <link rel="stylesheet" href="https://uicdn.toast.com/calendar/latest/toastui-calendar.min.css"/>
+    <link rel="stylesheet" href="screen.css"/>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
```

Comment:
initially - because fist "back" and "next" icon was suggested as the same onto card widget, that are glyph icons from bootstrap 3.0.
Although icon concretion change a little, buttons was already implemented using bootstrap at that point and works ok, so i don't decided to remove it